attestation-service/docs/policy.md

```diff
@@ -1,6 +1,29 @@
 # Attestation Policy (Developing)
 
 CoCo AS provides a flexible policy support based on Rego to facilitate the customized verification rules.
+The policies are used to general an EAR appraisal, which is part of the EAR token returned by the
```

Comment:
We might need to claim that different token type needs different policy formattings. The added part is for Ear